static/script.js

```diff
@@ -69,3 +70,26 @@ function enableAutoRefresh () { // eslint-disable-line no-unused-vars
     }
   }
 }
+
+function unfoldAll () { // eslint-disable-line no-unused-vars
+  window.setInterval(unfoldButton, 100)
+  $('#unfold-all-li').removeClass('d-none')
+  let allUnfoldIds = $('#refs_list').attr('content').replace(/[\[\]'"\ ]/gm, '').split(',').sort().toString() // eslint-disable-line
```

Comment:
You can split method calls to multiple line because JS terminates lines with semicolon not a new line.
This will work too and it is consider good practice for long chain of methods:
```javascript
let allUnfoldIds = $('#refs_list').attr('content')
                                  .replace(/[\[\]'"\ ]/gm, '')
                                  .split(',')
                                  .sort()
                                  .toString()
```
Side notes: To obtain the same behavior in Python, you have to terminate each line with backslash without any white characters between method and backslash.